sdk/eventgrid/azure-messaging-eventgrid/pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.azure</groupId>
   <artifactId>azure-messaging-eventgrid</artifactId>
-  <version>4.10.0-beta.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid;current} -->
+  <version>4.9.1</version> <!-- {x-version-update;com.azure:azure-messaging-eventgrid;current} -->
```

Comment:
We added new enum values (new APIs added) for this release. So, this must be a minor version update, not a patch version.